modules/virt-switching-displays.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
+
+:_content-type: PROCEDURE
+[id="virt-switching-displays_{context}"]
+= Switching between VM displays
+
+If your virtual machine (VM) has multiple displays, you can view each one by switching displays in the web console.
+
+.Prerequisites
+
+* The mediated device is configured in the `HyperConverged` custom resource and assigned to a VM.
+
+.Procedure
+
+. In the OpenShift Virtualization console, click *Workloads* -> *Virtualization* from the side menu.
+. Click the *Virtual Machines* tab.
```

Comment:
In 4.11 the Virtualization section is top menu (not under Workloads), the item is VirtualMachines
```suggestion
. In the OpenShift Virtualization console, click *Virtualization* from the side menu.
. Click the *VirtualMachines* tab.
```
![virtualmachinesMenuItem](https://user-images.githubusercontent.com/2181522/177036056-d31e5015-a6f3-4866-9ced-3c36c1e0b0af.png)